neo/app/controllers/Application.scala

```diff
@@ -72,6 +76,68 @@ class Application @Inject()(val wsClient: WSClient,
     }
   }
 
+  def loadCity() : List[JsValue] = {
+    val file = environment.getFile("/public/data/city.sample.shorten.json")
+    val stream = new FileInputStream(file)
+    val json = Json.parse(stream)
+    val values = (json \ "features").as[List[JsObject]]
+    var newValues = List[JsValue]()
```

Comment:
1. try to remove this newValues.
2. if not easy, you can first try to use `List.newBuilder`